bread/bread.py

```diff
@@ -308,67 +287,33 @@ class Bread(object):
     `{view}` (`browse`, `read`, etc.).
 
     """
-    base_browse_view_class = BrowseView
-    base_read_view_class = ReadView
-    base_edit_view_class = EditView
-    base_add_view_class = AddView
-    base_delete_view_class = DeleteView
+    browse_view = BrowseView
+    read_view = ReadView
+    edit_view = EditView
+    add_view = AddView
+    delete_view = DeleteView
 
     exclude = []  # Names of fields not to show
```

Comment:
`exclude` and `form_class` are a bit tricky since they're neither generic nor confined to one view. I have two suggestions on how to resolve this.

The first suggestion is to leave them where they are but change `exclude` to `exclude_from_form` to make it explicit. 

The second suggestion is to create a `FormViewsMixin` that contains only these 2 config params. The `REA` views (the ones that care about the form) would inherit from this mixin. `form_views_mixin` would be a new attribute on the `Bread` class alongside `browse_view`, `read_view`, etc.

I favor the second suggestion. The first suggestion is easier but doesn't do anything to fix two of the problems identified in the original issue, which are (a) the list of ctor params will become unmanageable as this package adds sophistication and (b) from a caller's POV, it's hard to tell which config param belongs to what.